.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
-## Pull Request Guidelines
+<!--
```

Comment:
I have tried to please everyone by making it more obvious what is an instruction (in the markdown comment) and saying below that the author must delete the comment.